test.js

```diff
@@ -46,6 +46,21 @@ describe('Cassandra Migration', function() {
                 assert.equal(executeSpy.args[0][0], expectedQuery.join(' '));
                 executeSpy.reset();
             });
+
+            it('A simple table with compression', function() {
+                db.createTable('users', {
+                    'name': 'varchar'
+                }, {
+                    'primary_key': 'name',
+                    'compression': "{'sstable_compression': 'DeflateCompressor', 'chunk_length_kb': 1024}"
```

Comment:
Would you like to have this option as a string literal or a JSON value which can be stringified inside the library?